src/components/card/card.stories.tsx

```diff
@@ -0,0 +1,339 @@
+import React from 'react'
+
+import { CardGroup } from './CardGroup/CardGroup'
+import { Card } from './Card/Card'
+import { Button } from '../Button/Button'
+
+export default {
+  title: 'Card',
+  parameters: {
+    info: `
+USWDS 2.0 Card components
+
+Source: https://designsystem.digital.gov/components/card/
+`,
+  },
+}
+
+const card = (
+  <Card
+    className="tablet:grid-col-4"
+    standardLayout="default"
+    header={
+      <header className="usa-card__header">
+        <h2 className="usa-card__heading">Card</h2>
+      </header>
+    }
+    footer={
+      <div className="usa-card__footer">
+        <Button type="button" className="usa-button">
+          Example Button
+        </Button>
+      </div>
+    }>
+    <div className="usa-card__body">
```

Comment:
What do we get from making this part of children props instead of explicitly assigning it to something like body?

From what I can tell the <header> and <div> elements are always necessary so should that be internal to the component instead of expecting them to be specified everytime with the right classes?  I could see Media maybe being it's own component but less so the header and footer content.

Reply:
I agree the body, header, footer style wrappers could be internal to the component (commented on that above). 

Not opinionated on the `children` question. I think its kind of 🎨 style thing here- often children are for the main body of  component that could have lots of different types of content.

Reply:
Yeah, I think this is a style preference.. personally I like using `props.children` where it makes sense semantically (which in this case I think it does) because I find it easier to parse:

```
<Card>
  <p>Here is my content!</p>
</Card>
```

than:
```
<Card
  body={<p>Here is my content!</p>} />
```

especially if it contains a lot of content.

Reply:
Resolved by making individual components for header, footer, body, and media. Card just renders children.